src/main/java/com/ibm/watson/developer_cloud/text_to_speech/v1/model/CustomVoiceModel.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.ibm.watson.developer_cloud.text_to_speech.v1.model;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * A customized voice model, that allows users to specify custom pronunciations
+ * for Waton's Text to Speech API.
+ * 
+ * @see <a href=
+ *      "http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/doc/text-to-speech/custom-intro.shtml">
+ *      Customization</a>
+ */
+public class CustomVoiceModel extends GenericModel {
+
+  @SerializedName("customization_id")
+  private String id;
+
+  private String name;
+
+  private String description;
+
+  private String language;
+
+  private String owner;
+
+  private Timestamp created;
```

Comment:
I considered this as well. If you give me further input, I can change it. The customizations API diverges from the other APIs by using unix timestamps instead of a date strings. So we could:

1. Use `Date` objects and adapt `DateDeserializer` to parse Unix timestamps: Downside: When serializing a CustomVoiceModel using GSON, the `DateSerializer` displays the Dates as date string and not as timestamp
2. Use `Date` objects and create a custom TypeAdapter for CustomVoiceModel only
3. Use `Timestamps`, which is a thin-layer on `Date`. This allows us to use an existing TypeAdapter. The methods `getCreated()` and `getLastModified()` still return Date objects, so the API user does not really see a difference.

What do you suggest?